packages/block-editor/src/components/floating-toolbar/index.native.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
```

Comment:
I believe it is added automatically and can be removed :)

Reply:
removed